winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+
+## Customize exported workbook and worksheet
+
+SfDataGrid exports to excel by using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). You can refer [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating workbook and sheet after exporting. 
+
+### Workbook
+SfDataGrid provides option to return [ExcelEngine](https://help.syncfusion.com/cr/file-formats/Syncfusion.XlsIO.ExcelEngine.html) from that you can get exported workbook. This allows you to protect, encrypt and add worksheet before saving. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Worksheet customization
+
+SfDataGrid provides support to export to already existing file or worksheet. 
+
+In the below code snippet, worksheet is created and passed to `ExportToExcel` method. In the same way, you can open already existing excel also using `XlsIO`. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+ExcelEngine excelEngine = new ExcelEngine();
+IWorkbook workBook = excelEngine.Excel.Workbooks.Create();
+dataGrid.ExportToExcel(dataGrid.View, options, workBook.Worksheets[0]);
+workBook.Version = ExcelVersion.Excel2013;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+Before saving workbook, you need to set the specific excel version by using `IWorkbook.Version` property. Here, you can directly manipulate the data in the worksheet. You can refer [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation) for more information.
+
+#### Setting borders
+
+You can set borders to excel cells by directly accessing worksheet after exporting data.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].UsedRange.BorderInside(ExcelLineStyle.Dash_dot, ExcelKnownColors.Black);
+workBook.Worksheets[0].UsedRange.BorderAround(ExcelLineStyle.Dash_dot, ExcelKnownColors.Black);
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-change-border-style.png" alt="Changing Border Style in Exported Excel for WinUI DataGrid" width="100%" Height="Auto"/>
+
+#### Enabling Filters
+
+You can show filters in exported worksheet by enabling filter for the exported range in the worksheet.
```

Comment:
Show filters in the exported worksheet by enabling a filter for the exported range in the worksheet.

Reply:
Content modified